### DISCUSSION TITLE
Custom field only numbers

### DISCUSSION THREAD

**Discussion by jferna30** [LiveHelperChat/livehelperchat#1990] [Q&A] [answered]
Good morning,

I am making a form and I would like one of the custom fields to be numeric but I can't find that option. Can you help me, please

**Comment by remdex** [comment, marked as answer]
Because there is no such option at the moment defining custom form fields. In the docs there is sample how to validate customer info through bot, just not at initial form submit.

**Comment by jferna30** [comment]
Thank you very much for your prompt response. It would be a great help if you had that option to add only numbers to a field

**Reply by remdex** [reply]
Well, you can always sponsor features you want 😃
